Xamarin.Forms.Platform.Android/Renderers/ShellToolbarTracker.cs

```diff
@@ -283,6 +286,27 @@ protected virtual async void UpdateLeftBarButtonItem(Context context, Toolbar to
 			}
 		}
 
+		protected virtual async Task SetDrawerArrowDrawableFromFlyoutIcon(Context context, ActionBarDrawerToggle actionBarDrawerToggle)
+		{
+			Element item = Page;
+			ImageSource icon = null;
+			while (!Application.IsApplicationOrNull(item))
+			{
+				if (item is IShellController shell)
+				{
+					icon = shell.FlyoutIcon;
+					if(icon != null)
+					{
+						var drawable = await context.GetFormsDrawable(icon);
+						actionBarDrawerToggle.DrawerArrowDrawable = new FlyoutIconDrawerDrawable(context, drawable);
```

Comment:
I think here need add a null check for `drawable` or add support null drawable in `FlyoutIconDrawerDrawable` class